crates/uv-cli/src/lib.rs

```diff
@@ -1014,6 +1014,38 @@ pub struct PipCompileArgs {
     #[arg(long, overrides_with("all_extras"), hide = true)]
     pub no_all_extras: bool,
 
+    /// Include dependencies from the specified dependency group.
+    ///
+    /// May be provided multiple times.
+    #[arg(long, conflicts_with("only_group"))]
+    pub group: Vec<GroupName>,
+
+    /// Exclude dependencies from the specified dependency group.
+    ///
+    /// May be provided multiple times.
+    #[arg(long)]
+    pub no_group: Vec<GroupName>,
+
+    /// Exclude dependencies from default groups.
+    ///
+    /// `--group` can be used to include specific groups.
+    #[arg(long, conflicts_with_all = ["no_group", "only_group"])]
+    pub no_default_groups: bool,
+
+    /// Only include dependencies from the specified dependency group.
+    ///
+    /// May be provided multiple times.
+    ///
+    /// The project itself will also be omitted.
```

Comment:
Presumably just copy/paste, but we should swap these lines (throughout).

Reply:
Done.